.circleci/config.yml

```
@@ -31,6 +39,12 @@ workflows:
       - publish_to_npm:
           filters:
             tags:
-              only: /^v.*/
+              only: /^v[0-9]*\.[0-9]*\.[0-9]*$/
+            branches:
+              ignore: /.*/
+      - publish_prerelease_to_npm:
+          filters:
+            tags:
+              only: /^v[0-9]*\.[0-9]*\.[0-9]*-/
```

Currently it would match `v..-`, `v1..-`, and `v1.1.-`, which is an invalid semver version. If we change the `*` to `+` then it would require atleast a single digit in each section, which is valid semver :-)
```suggestion
              only: /^v[0-9]+\.[0-9]+\.[0-9]+-/
```

---

Good catch, should have tested for that I forgot that `*` is match 0 or more times.